### PR TITLE
Added OpenMP course links

### PR DESCRIPTION
**Pull request by jhodrien**
Just added a couple to the guidance section, which at least means it'll pop up if you search for OpenMP.

I'm still on the look out for one with good exercises, so will add that later.

### CODE REVIEW

File: book/guidance/recommended_courses.md [hunk 1]
```diff
@@ -44,6 +45,11 @@ They are useful complements to your studies and the various [Training Courses](h
 - [Python Distilled](https://www.dabeaz.com/python-distilled/), David Beazley, 2021.
     - [Course](https://dabeaz-course.github.io/practical-python/)
 
+(openmp)=
+#### OpenMP
+- [Introduction to OpenMP](https://cvw.cac.cornell.edu/OpenMP/) from Cornell University
+- [Advanced OpenMP](http://www.archer.ac.uk/training/course-material/2019/06/AdvOpenMP-manch/L01-OpenMPBasics.pdf) from EPCC
```
Comment (NGRhodes) [inline book/guidance/recommended_courses.md:51]: Use https://  - the http:// is making my firefox complain.

**Review by NGRhodes — CHANGES_REQUESTED** [review]

File: book/guidance/recommended_courses.md (hunk 1 above)
Comment (jhodrien) [reply book/guidance/recommended_courses.md:51]: Hmm, but HTTPS has an expired certificate.  Maybe I need to find a better link to this material.

Comment (jhodrien) [reply book/guidance/recommended_courses.md:51]: https://www.archer2.ac.uk/training/courses/211005-advanced-openmp/

This appears to be the replacement course.  I'll review, and update the link to that if I like it.

Comment (jhodrien) [reply book/guidance/recommended_courses.md]: Fixed in 012c6d5

**Review by NGRhodes — APPROVED** [review]

**Pull request merged**